crates/topos/src/components/setup/commands/subnet.rs

```diff
@@ -0,0 +1,26 @@
+use clap::Args;
+use std::path::PathBuf;
+
+#[derive(Args, Debug)]
+#[command(about = "Install Polygon Edge node binary")]
+pub struct Subnet {
+    /// Installation directory path for Polygon Edge binary.
+    /// If not provided, Polygon Edge binary will be installed to the current directory
+    #[clap(long, env = "TOPOS_SETUP_SUBNET_PATH", default_value = ".")]
```

Comment:
maybe it will be more clear to refer to directory `dir` instead of `path` for this option
and maybe referring to polygon edge?

```suggestion
    #[clap(long, env = "TOPOS_POLYGON_EDGE_INSTALL_DIR", default_value = ".")]
```

```suggestion
    #[clap(long, env = "TOPOS_SETUP_POLYGON_EDGE_DIR", default_value = ".")]
```